docs/tutorial.md

```diff
@@ -10,17 +10,25 @@ In this tutorial, we'll go through each step of the process to get a basic Disco
 
 ## Requirements
 
-* A laptop or desktop running Ubuntu (or you can use a VM).
-* Juju and [Microk8s](https://juju.is/docs/olm/microk8s) installed. We’ll also want to make sure the ingress add-on is enabled, which we can do by running `microk8s enable ingress`.
+- A laptop or desktop running Ubuntu (or you can use a VM).
```

Comment:
I would suggestion changing this line to "A working station, e.g., a laptop, with amd64 architecture."